media/css/firefox/whatsnew/whatsnew-56.scss

```diff
@@ -0,0 +1,388 @@
+            }
+        }
+    }
+
+    @media #{$mq-desktop} {
+        .intro-wrapper {
+            padding-top: 0;
+
+            &:after {
+                height: 262px;
+                left: auto;
+                margin-left: 0;
+                right: 0;
+                width: 450px;
+            }
+        }
+
+        .intro-content {
+            text-align: left;
+            width: 45%;
+        }
+    }
+
+    @media #{$mq-desktop-wide} {
+        .intro-wrapper:after {
+            height: 319px;
+            width: 547px;
+        }
+    }
+}
+
+html[dir="rtl"] {
+    .intro {
+        .intro-wrapper:after {
+            margin-right: -150px;
+            right: 50%;
+        }
+
+        @media #{$mq-tablet} {
+            .intro-wrapper:after {
+                margin-right: -200px;
+            }
+        }
+
+        @media #{$mq-desktop} {
+            .intro-wrapper:after {
+                left: 0;
+                margin-right: 0;
+                right: auto;
+            }
+
+            .intro-content {
+                text-align: right;
+            }
+        }
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Features list
+
+.features-section {
+    @media #{$mq-tablet} {
+        padding-top: 40px;
+    }
+}
+
+.features-list {
+    padding-top: 240px;
+    position: relative;
+
+    li {
+        margin: 0 auto 40px;
+        max-width: 24em;
+        text-align: center;
+
+        &:first-child {
+            @include photon-horizontal-rule;
+            position: relative;
+        }
+    }
+
+    &:after {
+        @include at2x('/media/img/firefox/whatsnew_56/new-tab.png', 280px, 193px);
+        background-position: top right;
+        background-repeat: no-repeat;
+        content: '';
+        height: 203px;
+        margin-right: -140px;
+        position: absolute;
+        right: 50%;
+        top: 0;
+        width: 280px;
+    }
+
+    @media #{$mq-tablet} {
+        min-height: 435px;
+        padding-top: 0;
+        width: 50%;
+
+        li {
+            @include font-size-level3;
+            text-align: left;
+
+            &:first-child:before {
+                left: 0;
+                margin-left: 0;
+            }
+        }
+
+        &:after {
+            background-size: 600px 413px;
+            height: 435px;
+            margin-right: 0;
+            right: -660px;
+            width: 600px;
+        }
+    }
+
+    @media #{$mq-desktop} {
+        min-height: 673px;
+
+        li {
+            margin-bottom: 60px;
+        }
+
+        &:after {
+            background-size: 929px 639px;
+            height: 673px;
+            right: -1000px;
+            width: 929px;
+        }
+    }
+}
+
+html[dir="rtl"] .features-list {
+
+    li:first-child:before {
+        left: auto;
+        margin-right: -65px;
+        right: 50%;
+    }
+
+    &:after {
+        left: 50%;
+        margin-left: -140px;
+        right: auto;
+    }
```

Comment:
Noticed that this results in the part of the image showing the tab being offscreen to the left. Not sure if a big deal or not.

Reply:
The design is supposed to position the browser image slightly offset from the viewport, but yeah here the tab part is not visible. Ideally we would have a RTL browser screenshot here, but I don't think we'll likely get one in time. I don't think it's really a big deal though.